### ISSUE TITLE
MacOS X release not functional and not correctly packaged

### ISSUE THREAD

**Issue by AlexisDrogoul** [gama-platform/gama#1773]
The automatic release for MacOS X is not functional and is probably missing several components to become testable.

- [ ] It is missing the `Info.plist` file (the one present in `msi.gama.application`,  https://github.com/gama-platform/gama/blob/master/msi.gama.application/macosx/Info.plist), which should be present in `Gama.app/Contents/`. The info about the version (e.g. 1.7) should be included.
- [ ] It is missing the `Model.icns` and `icon.ins` files (https://github.com/gama-platform/gama/blob/master/msi.gama.application/macosx/Model.icns & https://github.com/gama-platform/gama/blob/master/msi.gama.application/icons/launcher_icons/icon.icns) in `/Contents/Resources`
- [ ] It is named `Eclipse.app` and not `Gama.app`

**Comment by AlexisDrogoul** [comment]
OK. I have made some tests this morning and I can report the following:

- when the bundle is correctly seeded with a correct Info.plist (the one in msi.gama.application/macosx) and a Resources folder containing the icons, both copied into Eclipse.app>Contents>, it works. 
- Tycho actually outputs the Info.plist but at the wrong location (it is in `Eclipse.app>Contents>Eclipse.app>Contents>Info.plist), so probably a copy script is wrong somewhere.
- So, basically, there are 3 things to do to have a working release:

1) Change the default name ('Eclipse.app') to Gama.app. When exporting with PDE, it is the name you have to give to the "root folder". Dont know if Tycho supports it or not.
2) Copy `msi.gama.application>Info.plist` into `Gama.app>Contents>`
3) Copy the folder `msi.gama.application>Resources` into `Gama.app>Contents>`

And it should be ok !

**Comment by AlexisDrogoul** [comment]
It seems that the root folder can be set using the directives here:

https://dentrassi.de/2014/09/09/creating-a-mac-os-app-bundle-with-maven-tycho/
